modelserver-theia/src/browser/modelserver-api.ts

```diff
@@ -41,12 +41,11 @@ export class DefaultModelServerApi implements ModelServerApi {
     @inject(ModelServerBackend) protected readonly modelServerBackend: ModelServerBackend;
     private restClient: RestClient;
 
-    initialize(): Promise<boolean> {
-        this.modelServerBackend.getLaunchOptions().then(options => {
+    async initialize(): Promise<boolean> {
```

Comment:
I think the `async` keyword is superfluous as we do not `await` anywhere and return a promise anyways.

Reply:
Right, I will remove the superfluous keywords